dandi/publish/models/version.py

```diff
@@ -118,3 +118,7 @@ def from_girder(cls, dandiset: Dandiset, client: GirderClient) -> Version:
         version = cls.from_girder_metadata(dandiset, metadata)
         version.save()
         return version
+
+    @classmethod
+    def published_count(cls):
```

Comment:
I realize this is counterintuitive, but I don't think this belongs on `Version`, in terms of how we've defined it. `Version.published_count` is *not* a returning count of published versions here.

I think it should probably go at `Dandiset.published_count`. As an implementation detail, it happens to start from `Version.object...`, but only because I'm not aware of a way that we can efficiently filter by a reverse relation from the perspective of a Dandiset.